test/e2e/create_test.go

```diff
@@ -589,3 +593,46 @@ func (tc *testContext) scaleDeployment(namespace, name, selector string, expecte
 	}
 	return nil
 }
+
+// checkMachineInstanceMetadata checks the instance metadata on the Windows machines and returns with success
+// after finding the first machine where the instance metadata endpoint is available. Otherwise, returns an error.
+// This check is platform specific, as the instance metadata endpoint is required in a subset of the supported
+// platforms (i.e. GCP).
+// The BYOH use case is also covered by this check, since both machine and BYOH instances runs on the same cloud
+// provider.
+func (tc *testContext) checkMachineInstanceMetadata(machines []mapi.Machine) (err error) {
```

Comment:
I dont see the value in this helper.
Its not testing the functionality of our code.
If the metadata endpoint isnt reachable, WMCO will fail to configure the VM.

Reply:
Correct. This test is redundant as WMCO won't be able to configure the Windows instance if the address of the instance metadata endpoint changes. 

Removed the test.

Reply:
I am fine with not having a test as long as WMCO failing to configuring the VM will point to an issue with the endpoint.